Dockerfile

```diff
@@ -1,15 +1,10 @@
 FROM ubuntu
 
 RUN apt update && apt install -y git make unzip curl g++ python3 pip wget
-COPY /build/requirements.txt ./
-RUN pip3 install -r requirements.txt
+RUN pip3 install Pillow Wave
 RUN git clone https://github.com/tensorflow/tflite-micro.git
 
 WORKDIR tflite-micro
-
-RUN make -f tensorflow/lite/micro/tools/make/Makefile hello_world_bin
```

Comment:
Why?

Reply:
I tied different versions of the Dockerfile to make the smallest possible Docker image, that would run the Hello World test and show the result on the terminal. The Hello World test did run without this line, so I removed it from this demonstration Dockerfile. In the final version of course we must include all the necessary files.